packages/frontend/src/features/metricsCatalog/components/MetricsTable.tsx

```diff
@@ -36,6 +36,10 @@ export const MetricsTable = () => {
         (state) => state.metricsCatalog.categoryFilters,
     );
 
+    const canManageTags = useAppSelector(
+        (state) => state.metricsCatalog.abilities.canManageTags,
+    );
```

Comment:
is this a new pattern that I don't know of? it feels strange to have abilities in the redux state.
fyi, im not asking for changes, just curious!

Reply:
This was introduced in this PR: https://github.com/lightdash/lightdash/pull/12310